src/site/stages/build/process-cms-exports/filters.js

```diff
@@ -19,6 +19,7 @@ const whitelists = {
     'changed',
     'moderation_state',
   ],
+  'paragraph-wysiwyg': ['field_wysiwyg'],
```

Comment:
Would it be valuable to refactor things so that the array of filtered values lives in the same file as the transformer for the entity?  We may be dealing with merge conflicts and a huge object otherwise.

Reply:
I think that makes sense. If we do that, we'll have to modify the way the transformers are automagically imported, but that should be easy enough. Probably warrants a separate PR, though.

Reply:
Yeah, I didn't want to do that kind of refactoring here, I just thought it was worth mentioning.